org.jacoco.core.test.validation.groovy/src/org/jacoco/core/test/validation/groovy/targets/GroovyDelegateClassTarget.groovy

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2019 Mountainminds GmbH & Co. KG and Contributors
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Jan Wloka - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.test.validation.groovy.targets
+
+/* This annotation generates on a field of type Date the following
+* - a suitable implementation of getTime()
+* - a suitable implementation of setTime(long)
+* - a suitable implementation of before(Date)
+* - a suitable implementation of after(Date)
+* - a suitable implementation of compareTo(Date)
+* - a suitable implementation of toInstant()
+* - a suitable implementation of getTarget()
+* - a suitable implementation of setTarget(Date)
+*/
+
+class GroovyDelegateClassTarget { // assertEmpty()
+
+    @groovy.lang.Delegate
```

Comment:
According to documentation of `groovy.lang.Delegate`

>  All public instance methods present in the delegate type **and not present in the owner class** will be added to owner class at compile time.

so I think that for completeness of test will be nice to add hand-written method that is present in `java.lang.Date` and verify that compiler won't mark it is as `Generated`.